quickwit/quickwit-cluster/src/cluster.rs

```diff
@@ -1095,7 +1091,7 @@ mod tests {
             );
             chitchat_guard.self_node_state().set(
                 format!("{INDEXING_TASK_PREFIX}01BX5ZZKBKACTAV9WEVGEMMVS1"),
-                "my_index:uid:my_source:3a,5".to_string(),
+                "my_index-uid-my_source:3,5".to_string(),
```

Comment:
I think the point was to test what happened in presence of a malformed entry

Reply:
It is still malformed. Unfortunately, any string is a valid shard ID now, so `3a` no longer breaks anything.